src/routes/(console)/project-[region]-[project]/sites/(components)/siteCard.svelte

```diff
@@ -46,7 +46,23 @@
     }
 
     function getFilePreview(fileId: string) {
-        return sdk.forConsoleIn(page.params.region).storage.getFileView('screenshots', fileId);
+        return sdk
+            .forConsoleIn(page.params.region)
+            .storage.getFilePreview(
```

Comment:
Let's also set resolutions for image, something reasonable. See what size image has on big screen, and double that (for retina displays). such size should be enough to have all details.